bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,329 @@
+        }
+        json_data = self.load_json(Path('bot', 'resources', 'halloween', 'spooknamerate.json'))  # load the JSON
+        self.first_names = json_data['first_names']  # get the first
+        self.last_names = json_data['last_names']  # and last
+        # the names are from https://www.mockaroo.com/
+        self.added_messages = ["Let's see if you win?", ":jack_o_lantern: SPOOKY :jack_o_lantern:",
+                               "If you got it, haunt it.", "TIME TO GET YOUR SPOOK ON! :skull:"]  # define some messages
+
+        self.first = True  # Is it the first time?
+        self.poll = False  # A variable to track whether the poll is running
+        self.channel_id = Channels.seasonalbot_commands  # Store the id
+        self.announce_word.start()  # Start announce word
+        self.word = None  # The name
+
+    @in_month(Month.OCTOBER)  # Lock to OCTOBER
+    @commands.group(name="spooknamerate", invoke_without_command=True)
+    async def spook_name_rate(self, ctx: Context) -> None:
+        """Get help on the Spook Name Rate game."""
+        # Send an embed with help on the command
+
+        help_embed = discord.Embed(
+            title="Spook Name Rate",
+            description=f"Help on the `{self.bot.command_prefix}spooknamerate` command",
+            color=Colours.soft_orange,
+        )
+
+        emoji_message = ""
+        for emoji in self.emojis_val:
+            emoji_message += f"- {emoji} {self.emojis_val.get(emoji)}\n"
+
+        help_embed.add_field(
+            name="How to play",  # display messages on HOW TO PLAY
+            value=f"""Everyday, the bot will post a random word, which you will need to spookify using your creativity.
+You can rate each message according to how scary it is
+At the end of the day, the author of the message with most reactions will be the winner of the day.
+On a scale of 1 to {len(self.emojis_val)}, the reactions order:
+{emoji_message}""",
+            inline=False
+        )
+
+        # command to add spookified name
+        help_embed.add_field(
+            name="How do I add my spookified word?",
+            value=f"Just simply type `{self.bot.command_prefix}spooknamerate add my word`",
+            inline=False
+        )
+
+        await ctx.send(embed=help_embed)
+
+    @spook_name_rate.command(name='list', aliases=["all", "entries"])
+    async def list_entries(self, ctx: Context) -> None:
+        """Send all the entries up till now in a single embed."""
+        await ctx.send(embed=await self.get_responses_list(final=False))  # Send a list of users
+
+    @spook_name_rate.command(name='word')
+    async def tell_word(self, ctx: Context) -> None:
+        """Tell the current random word (name)."""
+        if not self.poll:
+            return await ctx.send(f"The name is **{self.word}**")
+        else:
+            return await ctx.send(f"The word ~~is~~ was **{self.word}**. The poll has already started, so you cannot \
+add an entry.")
```

Comment:
I'd appreciate splitting it up into 2 strings, instead of using `\`

```suggestion
            return await ctx.send(
                f"The word ~~is~~ was **{self.word}**."
                " The poll has already started, so you cannot add an entry.")
``` 
(Same for others with `\` linebreaks)

Reply:
I changed that, thanks for the feedback!